euth/actions/emails.py

```diff
@@ -19,8 +21,10 @@ def notify_creator_on_create_action(action):
         'url': url,
     }
 
+    jsonDec = json.decoder.JSONDecoder()
+
     emails.send_email_with_template(
-        [action.target.creator.email], 'notify_creator', context)
+        jsonDec.decode(action.recipients), 'notify_creator', context)
```

Comment:
Yeah right, I also added an issue here: #495 (should that be changed in the send email with template function?)